api/tests/integration/infrastructure/repositories/jury-session-repository_test.js

```diff
@@ -49,11 +60,19 @@ describe('Integration | Repository | JurySession', function() {
           finalizedAt: null,
           resultsSentToPrescriberAt: null,
           publishedAt: null,
+          juryComment: 'Les mecs ils font des sessions de certif dans le jardin ??',
```

Comment:
c'est le moment de refacto et de remplacer le combo :
```
expect(expectedJurySession).to.be.an.instanceOf(JurySession);
expect(expectedJurySession).to.deep.equal(<something>);
```

par 
`expect(jurySession).to.deepEqualInstance(expectedJurySession);`


On remarquera d'ailleurs sur ce test que les modèles sont mal placés dans l'expect.
La syntaxe c'est :
`expect(valeurRetournéeParMaFonctionTestée).to.equal(valeurAttendue);`
et pas le contraire